homeassistant/components/http.py

```diff
@@ -443,3 +443,7 @@ def file(self, request, fil, mimetype=None):
 
         return self.Response(wrap_file(request.environ, fil),
                              mimetype=mimetype, direct_passthrough=True)
+
+    def options(self, request):
+        """Default handler for OPTIONS (necessary for CORS preflight)."""
+        return self.Response('', status=200)
```

Comment:
Well, we can, but the important bit is to return the CORS headers (which we will).

I can add the method options easily enough though.